_data/security.yml

```diff
@@ -26,6 +26,13 @@ websites:
       - totp
     doc: https://www.bitdefender.com/consumer/support/answer/1581/
 
+  - name: BitSight
+    url: https://service.bitsight.com
+    img: bitsight.png
+    tfa:
+      - totp
+    doc: https://help.bitsight.com/hc/en-us/articles/360057228514-BitSight-2-Factor-Authentication#bitsight_2fa
```

Comment:
I'd suggest we drop the doc link here, since it's not publicly available. The screenshot on the issue is sufficient for me.
```suggestion
```

Reply:
Sure thing.